build-aux/org.gnome.GTG.Devel.json

```
@@ -30,17 +30,23 @@
     "*.pyc",
     "*.pyo"
   ],
+  "add-extensions": {
+    "org.freedesktop.Sdk.Extension.texlive": {
+      "directory": "texlive",
+      "version": "23.08"
+    }
+  },
   "modules": [
-    "python3-requirements-dev.yaml",
+    "python3-requirements.yaml",
```

I have the texlive extension installed.
When running the flatpak I get the error message:
```
~$ flatpak run org.gnome.GTG.Devel
bwrap: Can't mkdir /app/texlive: Read-only file system
Fehler: ldconfig ist fehlgeschlagen, Exit-Status 256
```

```suggestion
    {
      "name": "extension-points",
      "buildsystem": "simple",
      "build-commands": [
        "mkdir ${FLATPAK_DEST}/texlive"
      ]
    },
    "python3-requirements.yaml",
```